book/website/community-handbook/consistency.md

```diff
@@ -0,0 +1,53 @@
+Achieving this goal is reasonably challenging. 
+However, this chapter provides support by curating and introducing a consistency checklist to point contributors towards what consistency checks to look for as they help make _The Turing Way_ consistent.
+
+(ch-consistency-requirements)=
+## Hard vs Soft Requirements
+
+Items in the consistency checklist are categorised into hard and soft requirements. 
+Hard requirements are essential consistency checks that a chapter must pass so that _The Turing Way_ builds without errors, and is readable and accessible by everyone.
+
+Soft requirements, on the other hand, are nice-to-have consistency checks that a chapter should pass. 
+These checks greatly enhance the overall look and feel of the book but can be safely ignored.
+
+An overview of these requirements is itemised in {ref}`below <ch-consistency-requirements-checklist>`. The rest of this chapter explains them in more detail and describes how they can be achieved.
+
+> Please note that these requirements are not exhaustive or definitive, and neither are the classifications rigid. 
+> If you identify more consistency issues that need to be addressed, join the conversation [here](https://github.com/alan-turing-institute/the-turing-way/issues/1174). 
+> There is also no order to the items in the checklist. 
+> No item is inherently more important than the other.
+
+(ch-consistency-requirements-checklist)=
+### Consistency Checklist
+
+> This checklist was developed based on the insights gained from _The Turing Way's_ participation in 2020's [Google Season of Docs](https://developers.google.com/season-of-docs) (GSoD) Programme.
+> The [Technical Writer](https://github.com/paulowoicho) selected to work on _The Turing Way_ helped make the book retrospectively consistent by creating guidelines, templates, and issues to facilitate new contributions.
+> The PRs and issues raised by the Technical Writer are tagged with the `gsod2020` label in _The Turing Way_ Github repo.
+
+- Hard Requirements
+  - Add labels to chapters, subchapters and images to enable cross-referencing.
+  - Convert all HTML formatting to Markdown.
+  - Convert image formatting from Markdown to `MyST`.
+  - Remove empty files from the `_toc.yml`.
+  - Ensure chapters follow a consistent structure.
+  - Ensure chapters are consistent with the Turing Way Style Guide.
+  - Ensure correct grammar and a consistent tone across the book.
+  - Ensure chapters use a consistent language.
+  - Add ALT text to images.
+  - Fix Markdown formatting of non-consecutive headers
+  - Remove ToCs from chapter body since one is auto-generated by `jupyter-books`.
+- Soft Requirements
+  - Ensure each subchapter has a good summary in their landing page.
+  - Split long chapters into smaller subchapters so they are modular.
+  - Ensure the names of chapters/subchapters map exactly to the left ToC.
+  - Ensure proper title-casing for headers.
```

Comment:
Add an example here, is it supposed to be the camel case or only the first letter capital